ui/raidboss/data/05-shb/raid/e11s.txt

```diff
@@ -52,35 +58,36 @@ hideall "--sync--"
 70.4 "Powder Mark" sync /:Fatebreaker:56A2:/
 77.9 "Turn Of The Heavens" sync /:Fatebreaker:566A:/
 89.3 "Burn Mark" sync /:Fatebreaker:56A3:/
-89.7 "Brightfire" sync /:Halo of Flame:566D:/
+89.7 "Brightfire" # sync /:Halo of Flame:566D:/
 
-94.4 "--middle--" sync /:Fatebreaker:5908:/
-103.9 "Burnt Strike" sync /:Fatebreaker:5654:/
+94.4 "--center--" sync /:Fatebreaker:5908:/
+103.9 "Burnt Strike" sync /:Fatebreaker:565[24]:/
 105.6 "Burnout/Blastburn" #sync /:Fatebreaker:565[35]:/
-117.6 "Bound Of Faith" sync /:Fatebreaker:5658:/
+117.6 "Bound Of Faith" sync /:Fatebreaker:565[8B]:/
 121.3 "Sinsmoke/Sinsmite" sync /:Fatebreaker:565[AD]:/
 
 # Dragon trio
-132.6 "Elemental Break" sync /:Fatebreaker:5666:/
-135.1 "Sinsmite" sync /:Fatebreaker:5667:/
+132.6 "Elemental Break" sync /:Fatebreaker:566[36]:/
+135.1 "Sinsmite/Sinsmoke" sync /:Fatebreaker:56(5A|57|65|67):/ window 10,10
 147.1 "Shifting Sky" sync /:Fatebreaker:567[56]:/
 149.5 "--untargetable--"
 154.7 "Ageless Serpent" sync /:Demi-Gukumatz:5687:/
 165.8 "Resonant Winds" sync /:Demi-Gukumatz:5689:/
-169.4 "Sinsmoke" sync /:Fatebreaker's image:5681:/
-169.4 "Sinsmite" sync /:Fatebreaker's image:5684:/
-169.8 "Bow Shock" sync /:Fatebreaker's image:5685:/
-174.7 "Burnout" sync /:Fatebreaker's image:567C:/
-178.0 "Blastburn" sync /:Fatebreaker's image:567A:/
+169.4 "Sinsmoke" sync /:Fatebreaker's Image:5681:/
+169.4 "Sinsmite" sync /:Fatebreaker's Image:5684:/
+169.8 "Bow Shock" sync /:Fatebreaker's Image:5685:/
+174.7 "Burnout" sync /:Fatebreaker's Image:567C:/
+178.0 "Blastburn" sync /:Fatebreaker's Image:567A:/
```

Comment:
Can you add the burnt strikes here from the adds? This is one thing fflogs didn't have, and similarly below?